### ISSUE TITLE
Breaking change version 3.2 -> 3.3

### ISSUE THREAD

**Issue by dorant** [gcovr/gcovr#137]
Hi!
There seems to be a bigger change when upgrading gcovr regarding paths/filters/exclusions:

In gcovr 3.2 the text output (and filter/exclusions) was absolute paths like:

```
File                                          Lines    Exec  Cover   Missing
------------------------------------------------------------------------------
/home/name/myrepo/file.cpp                         1       0     0%   31
```

but in 3.3 its relative to the build folder like

```
File                                          Lines    Exec  Cover   Missing
------------------------------------------------------------------------------
../../file.cpp                                     1       0     0%   31
```

In this case I'm using CMake, and my build folder where the gcovr is run from is:
/home/name/myrepo/build/variant/

This affects filters(-f) and exclusions (-e) that now also need to be in relative format,
like the hypothetical:

```
-f '/home/name/myrepo/file.*'
```

now needs to be stated:

```
-f '.*/file.*'
```

Is this expected?

Thanks for a great tool by the way!
BR/Bjorn

**Comment by whart222** [comment]
Bjorn:

This was not an expected change.  It should be fine to use both relative
and absolute paths for filters.

Can you create a ticket for this?  Even better would be pull request with a
new test case that illustrates this behavior.  I'm adding test cases for
things like this to avoid regressions in expected behavior.

**Comment by whart222** [comment]
I added a test case and I'm testing a fix for this.

**Comment by latk** [comment]
This seems to have been fixed in 90589892, though I'm quite sure this is not going to work on Windows. But that's a separate issue.
